### ISSUE TITLE
[Last-updated]: Add FE facing last updated with date feature

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#77836]
### Describe the problem

As a reviewer of the Platform website, I'd like to be able to know when a page was last updated and/or reviewed. 

### Who will benefit

Anyone reviewing the Platform website. 

### Describe your idea

Add the following towards the bottom of each page, which could be dynamically changed based on when the page in the CMS was published: 

Last updated: MMM DD, YYYY

### Provide evidence

It's been challenging to track the changes going on about the move from Universal GA to GA4. It looks like some pages on Platform were updated, but I was curious which ones were untouched. Having a last updated feature would allow site reviewers to know when a page was last modified. E.g., similar implementation on [VA DS Site](https://design.va.gov/components/).

### Platform Mission

- [x] Keeps these products and services available, stable, and secure.
- [X] Ensures they are consistent and meet high standards of quality in Veteran-centered user experience and technical approach.
- [x] Provides shared tools, software, support, and technical infrastructure to accelerate their development.
- [ ] Other

**Comment by jknipes** [comment]
Hi @benbrasso-agile6 thank you for flagging this. We'll take a look, and will work to get any updates reflected on the website.
We will provide further updates in this ticket. Please feel free to reach out if there are any questions.

**Comment by jknipes** [comment]
Hi @gopixelsgo this ticket has been moved into the Reviewed column on the Platform Feature Requests board

**Comment by jknipes** [comment]
There is a [Handy Timestamp](https://docs.stiltsoft.com/hmfc/cloud/handy-timestamp) macro for Confluence that appears to be publishable on Platform Website. cc: @laucon

**Comment by tayism** [comment]
Hi there @jknipes -- any recent updates here? Thanks!

**Comment by jknipes** [comment]
@tayism We are earnestly researching a way to add this feature to pages on Platform Website. Many macros and features available on Confluence do not render when published via Scroll Viewport on the Platform Website. We will be sure to update here when we are able to add this feature.

**Comment by tayism** [comment]
@jknipes Sounds good, and thanks! We appreciate the updates. SDT recently revamped the 'monitoring/cleaning' of the Feature Request Board, which helps ensure things are moving along (as they can).

**Comment by jknipes** [comment]
@tayism we have 2 new tools that are being implemented in the upcoming sprint, and I am wondering which will be of most benefit to solving this issue:

1. a date stamp will be added to the top of each newly edited page during deployment, which is Mon-Thurs. This refers to pages edited by Platform team members. There are also ~90 Product Directory documents that are updated automatically daily, and these would not get the date stamp.
2. Platform Content Team is also using a tool to create a list of the top 10-15 pages in each Platform Website space (Getting Started, Developer docs, Research and design, Analytics and insights, Collaboration cycle, Support) that have not been updated in 150 days or more.

**Comment by gopixelsgo** [comment]
Hi @jknipes. Following up on this to see if there were updates on this ticket that might move it out of Reviewed. If so, feel free to move it to the In Progress column or wherever appropriate. Thanks!

**Comment by jknipes** [comment]
@gopixelsgo we have implemented a process whereby anyone creating or editing documentation for Platform Website must add a date stamp with the current date at the top of the page. This work is reflected in the tickets on Epic [79850](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/79850).
Additionally, we have created the [Keep It Up To Date page  ](https://vfs.atlassian.net/wiki/spaces/CIT/pages/3061743626/Keep+It+Up+To+Date+-+6+Spaces+on+Platform+Website) in Confluence, which will be announced monthly in a reminder message on Slack. This page informs users via the Keep It Up To Date application of the most outdate pages on Platform Website, giving teams an opportunity to go directly to the page and review the content for possible updates needed.
